model/schema/core.yaml

```diff
@@ -171,12 +171,12 @@ classes:
       A-Team curation UI Controlled Vocabulary Terms Table.
     slots:
       - free_text
-      - note_type
+      - note_type_name
```

Comment:
This should still be `note_type` with a range of `VocabularyTerm`.  The `NoteDTO` class  uses `note_type_name`